### PR TITLE
Make input importing  easier

### PR DESCRIPTION
**Pull request by ericls**

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+    last_frame = inspect.stack()[1]
+    module = inspect.getmodule(last_frame[0])
+    input_path = Path(module.__file__).parent / "input.txt"
+    with open(input_path, "r") as f:
+        content = f.read().strip()
+        if split_lines:
+            content = content.splitlines()
+        if line_transform is not None:
+            content = [line_transform(line) for line in content]
```
Comment (p7g) [inline utils.py]: What do you think of this? I think maybe `line_transform` should imply `split_lines`

Not 100% sure about always calling `.strip()` on each line though 🤔 

```suggestion
        if split_lines or line_transform:
            content = content.splitlines()
            if line_transform is not None:
                content = [line_transform(line.strip()) for line in content]
```

**Review by p7g — COMMENTED** [review]
Very nice, I was just thinking it would be great to have something like this.

Maybe this module could be `lib/input.py`? Just to keep the root a little cleaner

File: utils.py (hunk 1 above)
Comment (ericls) [reply utils.py]: that's a good idea, one less args to pass. 

In terms of strip() I think it's better to make it configurable just in case...

Comment (p7g) [reply utils.py]: The `.strip()` can probably just be in `line_transform` tbh

**Pull request merged**